x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/component/scan_usage_results.tsx

```diff
@@ -37,13 +38,19 @@ export const ScanUsageResults: React.FC<ScanUsageResultsProps> = ({
   onIgnoreWarningCheckboxChange,
 }) => {
   const {
-    services: { application },
+    services: { application, serverless },
   } = useKibana();
   const handleNavigateToIndexManagement = () => {
-    application?.navigateToApp(ENTERPRISE_SEARCH_CONTENT_APP_ID, {
-      path: 'search_indices',
-      openInNewTab: true,
-    });
+    if (serverless) {
+      application?.navigateToApp(SEARCH_INDICES, {
```

Comment:
I guess this is ok, `SEARCH_INDICES` is actually doing a redirect to AppEx IndexManagement though so you could just link to that directly.

But again longer term we should have Locators for this too. so if we create a github issue mention this usage as well.

Reply:
Yes, I have a ticket to handle this locator issue later.
 Is this problem statement correct?
"To ensure if a plugin provides URL based on the platform (ES3, ESS) and the consumer plugin will have to call `getUrl` function and receive the appropriate URL."

Reply:
yea that should be good. you could also use the same pattern for ingest pipelines navigation since they already define a [locator](https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/x-pack/plugins/ingest_pipelines/public/locator.ts#L59). so you could handle refactoring them at the same time.